chat-plugins/mafia.js

```diff
@@ -72,6 +74,97 @@ class MafiaPlayer extends Rooms.RoomGamePlayer {
 	}
 }
 
+// Parses a single role into an object
+function parseRole(roleString) {
```

Comment:
Why not as part of the class?

Reply:
I separated it out to let you get information on a role by itself, see 1703. Vaguely remember someone asking for it as a feature.